### PR TITLE
mobile: improve android push notif delivery time

### PR DESCRIPTION
**Pull request by sduthil**
Why:

* the time_to_live = 0 header makes FCM not store the notification and
  deliver it immediately if an app is registered for it else drop the
  notification

### CODE REVIEW

**Review by manuquentin — DISMISSED** [review]

**Comment by sduthil** [comment]
Auto-merging, since only tests were missing after approval.

**Pull request merged**
